frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.tsx

```diff
@@ -169,7 +169,17 @@ class DatasetActionView extends React.PureComponent<Props, State> {
           onClick={() =>
             Modal.error({
               title: "Cannot load this dataset",
-              content: dataset.status,
+              content: (
+                <div>
+                  <p>{dataset.status}</p>
+                  {dataset.status === "Deleted by user." ? (
+                    <p>
+                      Even though, this dataset was deleted by a user, it is still shown here,
+                      because it is referenced by at least one annotation.
```

Comment:
Right now it should be "it **was** referenced" since it will not deleted if all associated annotations are deleted eventually. Follow up issue?

Reply:
you're right. "was" is clearer. I changed it. A follow-up issue for automatically "completing the deletion" once all annotations are deleted is not necessary imo.